### DISCUSSION TITLE
Following Instructions

### DISCUSSION THREAD

**Discussion by drteresavasquez** [nss-evening-cohort-14/evening-client-side#113] [Discuss a Topic]
# Intro
As a developer, it is important that you follow directions and instructions that are given to you and ask questions for clarity. Please complete the following exercise to show your skillz 🌟 

## Instructions
1. Read/Watch all the instructions
2. Post a gif under this thread https://nashvillesoftware.slack.com/archives/C0157TSDCSC/p1613582403434800
3. Put a heart under at least 3 gifs
4.  In the cohort slack channel, post "I am here"
5. Start a new webpack project called "Follow Instructions" do all the setup
6. Watch the following video: https://vimeo.com/513504898
7. You are all done! Congrats!

**Comment by drteresavasquez** [comment]
### Completed:
- [X] @caseywalker 
- [X] @tsekeres 
- [X] @SaraSchoonover 
- [X] @katherinevfry 
- [X] @jrobinson0529 
- [X] @cmeffley 
- [X] @robertncole1 
- [X] @thedigitalmenagerie

**Reply by fimoefive** [reply]
I got it!!

**Comment by Gabrielle-Tobermann** [comment]
Got it

**Comment by Mitchell-Crumbley** [comment]
I got it

**Comment by GonzalesMatthew** [comment]
I got it

**Comment by JuanDavila1101** [comment]
I got it

**Comment by chiestroud** [comment]
I got it

**Comment by jimguistical** [comment]
I got it.

**Comment by nashvegasnate** [comment]
Got it.

**Comment by Greenfin17** [comment]
Got it.

**Comment by danicrosby** [comment]
I got it.

**Comment by BenJarrett** [comment]
I got it!

**Comment by seancrossettie** [comment]
I got it.

**Comment by nathanmartin5937** [comment]
Got it!

**Comment by HunterJuneau** [comment]
I got it.
